setup.py

```diff
@@ -21,7 +21,7 @@
     'docs': [
         # pin sphinx to match what RTD uses:
         # https://github.com/readthedocs/readthedocs.org/blob/ecac31de54bbb2c100f933e86eb22b0f4389ba84/requirements/pip.txt#L16
-        'sphinx',
+        'sphinx>-5.0',
```

Comment:
Re: Docs error, we ran into the exact same issue with SCT, see: https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4340

In order to get `sphinx==5.x`, we had to loosen some other docs-related dependencies to avoid conflicts. (Leaving it unpinned should be OK unless something else is holding it back?)

Reply:
I'll try that thanks!

Reply:
It looks like unpinning the other dependencies resulted in the following packages getting installed:

```
sphinx-autodoc-typehints-1.25.3 
sphinx-jinja2-compat-0.2.0.post1 
sphinx-jsonschema-1.19.1 
sphinx-prompt-1.8.0 
sphinx-tabs-3.4.5 
sphinx-toolbox-3.5.0 
sphinx_rtd_theme-2.0.0 
sphinxcontrib-jquery-4.1
```

With our previous pins being:

```
'sphinx-tabs==3.2.0',
'sphinx-toolbox==2.15.2',
'sphinx-jsonschema~=1.16',
```

The last one (`'sphinx-jsonschema~=1.16'`) is compatible with `1.19.1`, but the other two were definitely quite out of date. 

I do notice that `sphinx-toolbox` had a major version upgrade, though. So, I would want to make sure that nothing major has changed in the docs before OK'ing this. I'll look over the changelog now. :)

Reply:
It looks like `sphinx-toolbox` got sphinx 5.x support in [version 3.2.0](https://sphinx-toolbox.readthedocs.io/en/stable/changelog.html#id3), so upgrading major versions was going to be necessary regardless. Likewise, `sphinx-tabs` added support for sphinx 5.x in [version 3.4.0](https://github.com/executablebooks/sphinx-tabs/blob/master/CHANGELOG.md#340---2022-06-26).

There's nothing that jumps out at me in either changelog as anything to worry about?

- `sphinx-tabs`: Tabs on pages such as https://ivado-medical-imaging--1313.org.readthedocs.build/installation.html seem to render OK.
- `sphinx-toolbox`: We only use the `collapse` utility from sphinx-toolbox, and the collapsible text in https://ivado-medical-imaging--1313.org.readthedocs.build/tutorials/cascaded_architecture.html also seems to render OK.